Formula/hyperkit.rb

```diff
@@ -0,0 +1,85 @@
+class Hyperkit < Formula
+  desc "Toolkit for embedding hypervisor capabilities in your application"
+  homepage "https://github.com/moby/hyperkit"
+  url "https://github.com/moby/hyperkit/archive/v0.20180403.tar.gz"
+  sha256 "e2739b034f20d9437696de48ace42600f55b7213292ec255032b2ef55f508297"
+
+  depends_on "aspcud" => :build
+  depends_on "ocaml" => :build
+  depends_on "opam" => :build
+  depends_on :xcode => ["9.0", :build]
+
+  depends_on "libev"
+
+  resource "tinycorelinux" do
+    url "https://dl.bintray.com/markeissler/homebrew/hyperkit-kernel/tinycorelinux_8.x.tar.gz"
+    sha256 "560c1d2d3a0f12f9b1200eec57ca5c1d107cf4823d3880e09505fcd9cd39141a"
+  end
+
+  def install
+    system "opam", "init", "--no-setup"
+    opam_dir = "#{buildpath}/.brew_home/.opam"
+    ENV["CAML_LD_LIBRARY_PATH"] = "#{opam_dir}/system/lib/stublibs:/usr/local/lib/ocaml/stublibs"
+    ENV["OPAMUTF8MSGS"] = "1"
+    ENV["PERL5LIB"] = "#{opam_dir}/system/lib/perl5"
+    ENV["OCAML_TOPLEVEL_PATH"] = "#{opam_dir}/system/lib/toplevel"
+    ENV.prepend_path "PATH", "#{opam_dir}/system/bin"
+
+    inreplace "#{opam_dir}/compilers/4.05.0/4.05.0/4.05.0.comp",
+      '["./configure"', '["./configure" "-no-graph"' # Avoid X11
+
+    ENV.deparallelize { system "opam", "switch", "4.05.0" }
```

Comment:
Is this an upstream issue or an `opam` issue?

Reply:
I replicate this from `docker-machine-driver-xhyve` Formula which also need `opam` to build with `qcow` support. It comes from https://github.com/Homebrew/homebrew-core/commit/66673949783511c0283bd1ce16f8c6893d5230c7 by @ilovezfs